crates/nu-cli/src/commands/alias.rs

```diff
@@ -59,19 +64,58 @@ impl WholeStreamCommand for Alias {
     }
 }
 
+// <<<<<<< HEAD
+// pub fn alias(alias_args: AliasArgs, ctx: RunnableContext) -> Result<OutputStream, ShellError> {
+// =======
 pub fn alias(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStream, ShellError> {
     let registry = registry.clone();
     let stream = async_stream! {
-        let (AliasArgs { name, args: list, block }, _) = args.process(&registry).await?;
-        let mut args: Vec<String> = vec![];
+        let mut raw_input = args.raw_input.clone();
+        let (AliasArgs { name, args: list, block, save}, ctx) = args.process(&registry).await?;
+        let mut processed_args: Vec<String> = vec![];
+
+        if let Some(true) = save {
+            let mut result = crate::data::config::read(name.clone().tag, &None)?;
+
+            // process the alias to remove the --save flag
+            let left_brace = raw_input.find('{').unwrap();
+            let right_brace = raw_input.rfind('}').unwrap();
```

Comment:
I'm surprised that clippy let you `.unwrap()` here. We went through a phase a while back to see if we could remove unwraps and use things that don't panic as easily.  You could use something like `if left Ok(v) = ...` instead